include/library/spdm_transport_tcp_lib.h

```diff
@@ -0,0 +1,169 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2025 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+#ifndef SPDM_TCP_TRANSPORT_LIB_H
+#define SPDM_TCP_TRANSPORT_LIB_H
+
+#include "library/spdm_common_lib.h"
+
+#define LIBSPDM_TCP_ALIGNMENT 4
+
+#define LIBSPDM_TCP_SEQUENCE_NUMBER_COUNT 0
+#define LIBSPDM_TCP_MAX_RANDOM_NUMBER_COUNT 0
+
+/* Required sender/receive buffer in device io.
+ * +-------+--------+---------------------------+------+--+------+---+--------+-----+
+ * | TYPE  |TransHdr|      EncryptionHeader     |AppHdr|  |Random|MAC|AlignPad|FINAL|
+ * |       |        |SessionId|SeqNum|Len|AppLen|      |  |      |   |        |     |
+ * +-------+--------+---------------------------+------+  +------+---+--------+-----+
+ * |  TCP  |    4   |    4    |   0  | 2 |   2  |   0  |  |   0  | 16|   3    |  31 |
+ * +-------+--------+---------------------------+------+--+------+---+--------+-----+
+ */
+#define LIBSPDM_TCP_TRANSPORT_ADDITIONAL_SIZE        (12 + \
+                                                      LIBSPDM_PCI_DOE_SEQUENCE_NUMBER_COUNT + \
+                                                      LIBSPDM_PCI_DOE_MAX_RANDOM_NUMBER_COUNT + \
+                                                      LIBSPDM_MAX_AEAD_TAG_SIZE + \
+                                                      (LIBSPDM_PCI_DOE_ALIGNMENT - 1))
+
+#pragma pack(1)
+
+typedef struct {
+    uint16_t payload_length;
+    uint8_t reserved;
+    uint8_t message_type;
+} tcp_spdm_binding_header_t;
```

Comment:
this is defined in spec. please move to industry standard header file.
Not here.

Reply:
done.